report/03_methodology/02_architecture.md

```diff
@@ -0,0 +1,38 @@
+## Architecture
+
+In this part of the report, the architecture of the application is described in more detail.
+
+### Architecture overview
+
+Overall, the application is divided into two completely independent parts. 
+On one hand, there is the stand-alone client app, which contains the self-test and a questionnaire. 
+On the other hand, there is the server application that gives an export functionality for the collected research data.
+
+#### Stand-alone client application
+
+The client app includes the self-test, which is supposed to give reliable data about the test person several times a day after it has been carried out. 
+The *Psychomotor vigilance task* and *Spatial span task* are two tasks included in the self-test. 
+There is also a questionnaire, which is supposed to record the current condition of the subject at longer intervals. 
+This happens via a certain number of questions with four choices each. 
+There should also be a notification for the self-test, which encourages the user up to three times a day to carry out the tests. 
+In addition, the current mood and current activity of the test person is saved before each self-test so that the data can also be interpreted reliably. 
```

Comment:
```suggestion
In addition to that, the current mood and current activity of the test person is saved before each self-test so that the data can also be interpreted reliably. 
```